titiler/settings.py

```diff
@@ -0,0 +1,10 @@
+"""Application environment variables"""
+
+from starlette.config import Config
+
+config = Config()
+
+DEFAULT_MOSAIC_BACKEND = config("DEFAULT_MOSAIC_BACKEND", cast=str, default="s3://")
+
+# TODO: Remove default (default is currently set to pass tests)
+DEFAULT_MOSAIC_HOST = config("DEFAULT_MOSAIC_HOST", cast=str, default="placeholder")
```

Comment:
can't you use fake value in https://github.com/developmentseed/titiler/blob/master/tests/conftest.py#L22 for testing ?

Reply:
also those should only be required if we use the mosaic ?